.rubocop_todo.yml

```diff
@@ -0,0 +1,575 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config --exclude-limit 0`
+# on 2021-04-21 06:50:09 UTC using RuboCop version 1.12.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
This file is generated automatically. So no need to review unless you are interested in seeing what rules are added😄